docs/architecture.md

```diff
@@ -1,280 +1,46 @@
 # Architecture
 
-This document describes the internal components that the Flow Data Provisioning Service is constituted of, as well as
-the API it exposes.
+The following document gives a brief overview of different parts of the Flow DPS architecture.
 
-**Table of Contents**
+## Flow DPS Indexer
 
-1. [Chain](#chain)
-    1. [Disk Chain](#disk-chain)
-2. [Feeder](#feeder)
-    1. [Disk Feeder](#disk-feeder)
-3. [Mapping State Machine](#mapping-state-machine)
-4. [Index](#index)
-    1. [Database Schema](#database-schema)
-        1. [First Height](#first-height)
-        2. [Last Height](#last-height)
-        3. [Header Index](#header-index)
-        4. [Commit Index](#commit-index)
-        5. [Events Index](#events-index)
-        6. [Path Deltas Index](#path-deltas-index)
-        7. [Block Height Index](#block-height-index)
-        8. [Transaction Records](#transaction-records)
-        9. [Block Transaction Index](#block-transaction-index)
-        10. [Collection Index](#collection-index)
-        11. [Collection Guarantee Index](#collection-index)
-        12. [Block Collection Index](#block-collection-index)
-        13. [Transaction Result Index](#transaction-result-index)
-        14. [Seals Index](#seals-index)
-        15. [Block Seals Index](#block-seals-index)
-        16. [Transaction Height Index](#transaction-height-index)
+The Flow DPS Indexer can only index past sporks, contrary to the [Flow DPS Live](#flow-dps-live) binary which indexes live sporks.
+In order to build its index, it uses the spork's root checkpoint, and goes through the protocol state database and write-ahead logs from its nodes to index the state of the blockchain throughout its history.
+It does not run the DPS API on its own, so the API must be started using the produced index, once the indexing operation is complete.
 
-## Chain
+[![Flow DPS Indexer](./svg/flow_dps_indexer.svg)](./svg/flow_dps_indexer.svg)
```

Comment:
For the `flow-dps-index`, in the illustration, the arrow goes from DPS API to the index:
![image](https://user-images.githubusercontent.com/20494379/135070339-b8ad5bbe-bd8f-4c3a-a04a-447222c55c80.png)

While for the DPS live indexer and the DPS APIs, it goes in the other direction.

![image](https://user-images.githubusercontent.com/20494379/135070573-6d5fe915-12da-4810-8384-6fa717273d10.png)

Reply:
Yeah, we should keep the arrows flowing in the same direction.